calendar-app/src/App.test.tsx

```diff
@@ -1,9 +1,72 @@
-import React from "react";
 import { render, screen } from "@testing-library/react";
 import App from "./pages/App";
+import { EventProvider } from "./utils/EventContext";
+import { Provider } from "react-redux";
+import { store } from "./store";
+import getGMT from "./utils/getGMT";
+import userEvent from "@testing-library/user-event";
+import { StoredEvent } from "./utils/types";
+import Event from "./pages/calendar/Event";
 
-test("renders learn react link", () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+describe("TimezoneGetter", () => {
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+  test("returns GMT +03 for Vilnius timezone offset", () => {
+    jest.spyOn(Date.prototype, "getTimezoneOffset").mockReturnValue(-180);
+    expect(getGMT()).toBe("GMT +03");
+  });
+
+  test("returns GMT 00 for UTC timezone offset", () => {
+    jest.spyOn(Date.prototype, "getTimezoneOffset").mockReturnValue(0);
+    expect(getGMT()).toBe("GMT 00");
+  });
+
+  test("returns GMT -04 for UTC -4 timezone offset", () => {
+    jest.spyOn(Date.prototype, "getTimezoneOffset").mockReturnValue(240);
+    expect(getGMT()).toBe("GMT -04");
+  });
+});
+
+describe("UI interactions", () => {
```

Comment:
```suggestion
describe("App", () => {
```
I would name the tests firstly by the component and if there are a lot of component tests, later split it to additional sub-suites